src/renderer/widgets/WalletDetails/ui/ProxiesList.tsx

```diff
@@ -0,0 +1,133 @@
+import { groupBy } from 'lodash';
+import { useUnit } from 'effector-react';
+
+import { cnTw, copyToClipboard, toAddress } from '@shared/lib/utils';
+import { ChainTitle } from '@entities/chain';
+import { useI18n } from '@app/providers';
+import { Accordion, DropdownIconButton, FootnoteText, HelpText, IconButton } from '@shared/ui';
+import type { Chain, ID, ProxyAccount } from '@shared/core';
+import { EmptyProxyList, ProxyAccount as ProxyAccountComponent, proxyModel } from '@entities/proxy';
+import { chainsService } from '@entities/network';
+import { AssetBalance } from '@entities/asset';
+import { DropdownIconButtonOption } from '@shared/ui/Dropdowns/common/types';
+import { ExplorersPopover } from '@entities/wallet';
+import { walletProviderModel } from '../model/wallet-provider-model';
+
+type Props = {
+  walletId: ID;
+  canCreateProxy?: boolean;
+  className?: string;
+};
+
+export const ProxiesList = ({ walletId, className, canCreateProxy = true }: Props) => {
+  const { t } = useI18n();
+
+  const proxyAccounts = useUnit(walletProviderModel.$proxyAccounts);
+  const proxyChainGroups = useUnit(proxyModel.$proxyChainGroupStore)[walletId];
+
+  if (!proxyAccounts || !proxyAccounts.length) return <EmptyProxyList className={className} />;
```

Comment:
`ProxiesList` responsibility is to display the list of proxies
It must know nothing about `EmptyList`
The condition that controls what component to display must be before `ProxiesList` or `EmptyProxiesList`